ajax/libs/jeditable.js/package.json

```diff
@@ -1,17 +1,30 @@
 {
   "name": "jeditable.js",
-  "filename": "jeditable.min.js",
-  "version": "1.7.3",
-  "author": "Mika Tuupola",
-  "description": ". With few lines of JavaScript code I allow you to click and edit the content of different html elements. I am based on Dylan Verheul’s editable.",
+  "filename": "jquery.jeditable.min.js",
+  "version": "1.8.0",
+  "author": {
+    "name": "Mika Tuupola",
+    "email": "tuupola@appelsiini.net",
+    "web": "http://www.appelsiini.net/projects/jeditable"
```

Comment:
This is a lib's website.
Let's remove this line.
Let's modify this field as the same with upstream is a good way.
`"author": "Mika Tuupola <tuupola@appelsiini.net>",`
https://github.com/NicolasCARPi/jquery_jeditable/blob/master/package.json#L28

Thank you.